scripts/trilinos-integration/white_run_jenkins_script_cuda-pascal

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -el
+ulimit -c 0
+
+KOKKOSKERNELS_BRANCH=$1
+TRILINOS_UPDATE_BRANCH=$2
+TRILINOS_PRISTINE_BRANCH=$3
+KOKKOS_UPDATE_BRANCH=$4
+
```

Comment:
Does this needs a separate file ? I thought this would an i/p parameter like the four parameters here. These are few line changes, right ?

Reply:
We could make it a single file with extra parameters, but this seemed the easier path to me.

Reply:
Ok, go ahead